src/index.js

```diff
@@ -16,8 +16,8 @@ import { loadComponent } from "./loadComponent";
 
 const registerWebComponents = function() {
     loadComponent("protvista-manager", ProtvistaManager);
-    loadComponent("protvista-sequence", ProtvistaSequence);
     loadComponent("protvista-filter", ProtvistaFilter);
+    loadComponent("protvista-pdb-sequence", ProtvistaPdbSequence);
```

Comment:
Does it make sense to keep the old protvista-sequence while adding the new protvista-pdb-sequence?

Reply:
I think I'm not following you... 😓

Reply:
In the original code, there was a `loadComponent("protvista-sequence", ProtvistaSequence)` which was removed. Do we want to keep that?